ci/travis/before_install-linux.sh

```diff
@@ -37,7 +37,7 @@ if [ $CC = 'clang' ]; then
     export CXX='clang++'
 fi
 
-export PIP_USER="--user"
+export PIP_USER=--user
```

Comment:
@rhyolight, what's the difference? Quotes or no quotes, the end result should be the same in this case.